Essentials/src/main/java/com/earth2me/essentials/Trade.java

```diff
@@ -193,23 +191,23 @@ public void isAffordableFor(final IUser user, final CompletableFuture<Boolean> f
         }
 
         if (getMoney() != null && getMoney().signum() > 0 && !user.canAfford(getMoney())) {
-            future.completeExceptionally(new ChargeException(tl("notEnoughMoney", NumberUtil.displayCurrency(getMoney(), ess))));
+            future.completeExceptionally(new ChargeException(user.playerTl("notEnoughMoney", NumberUtil.displayCurrency(getMoney(), ess))));
```

Comment:
We now have `TranslatableException`. Do we similarly want `ChargeException`s to also take in the key and be translated at display time?

Reply:
TranslatableException is made for running within the command system, and futures are scary here and I don't want to mix things up.